packages/forma-36-react-components/src/components/Notification/NotificationsManager.tsx

```diff
@@ -18,7 +18,7 @@ export interface Notification {
   id: string | number;
   text: string;
   close: Function;
-  duration: number;
+  duration: number | undefined;
```

Comment:
Not sure if this is a terrible idea or not, but it got around TypeScript complaining